modules/ROOT/pages/apikit-asyncapi-module-reference.adoc

```diff
@@ -295,41 +341,77 @@ How much the buffer size expands if it exceeds its initial size. A value of zero
+| Leader Epoch | Number | The epoch of the leader broker at the time the message is sent | None | No
+| Log Append Timestamp | DateTime | The timestamp when the message is appended to the log | None | No
 | Offset a| Number | The message offset in the Kafka topic partition | None | No
-| Partition a| Number | The partition within the Kafka topic where the message is stored | None | No
-| Protocol a| String | The protocol the Kafka server uses | None | No
-| Server Name a| String | The name of the Kafka server that processes the message | None | Yes
-|======================
+| Partition | Number | The partition within the Kafka topic where the message is stored | None | No
+| Protocol | String | The protocol the Kafka server uses | None | No
+| Server Name | String | The name of the Kafka server that processes the message | None | Yes
+|===
 
 [[AnypointMqMessageAttributes]]
 === Anypoint MQ Message Attributes
 
 [cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
-|======================
+|===
 | Field | Type | Description | Default Value | Required
-| Ack Token a| String | Token used for acknowledging the receipt of a message | None | No
-| Channel Name a| String | The name of the Anypoint MQ channel where the message is published or consumed | None | Yes
-| Channel Parameters a| Object | Additional parameters or settings specific to the channel | None | No
-| Content Type a| String | The MIME type of the message content | None | No
-| Delivery Count a| Number | The number of times the message is delivered | None | No
-| Headers a| Object | Metadata associated with the message, typically key-value pairs | None | No
-| Message Id a| String | Unique identifier for the message | None | Yes
-| Properties a| Object | Custom properties associated with the message | None | No
-| Protocol a| String | The protocol the Anypoint MQ server uses | None | No
-| Server Name a| String | The name of the Anypoint MQ server that processes the message | None | Yes
-|======================
+| Ack Token | String | Token used for acknowledging the receipt of a message | None | No
+| Channel Name | String | The name of the Anypoint MQ channel where the message is published or consumed | None | Yes
+| Channel Parameters | Object | Additional parameters or settings specific to the channel | None | No
+| Content Type | String | The MIME type of the message content | None | No
+| Delivery Count | Number | The number of times the message is delivered | None | No
+| Headers | Object | Metadata associated with the message, typically key-value pairs | None | No
+| Message Id | String | Unique identifier for the message | None | Yes
+| Properties | Object | Custom properties associated with the message | None | No
+| Protocol | String | The protocol the Anypoint MQ server uses | None | No
+| Server Name | String | The name of the Anypoint MQ server that processes the message | None | Yes
+|===
+
+[[SolaceMessageAttributes]]
```

Comment:
@ashutoshsharmasf please review these parameters

Reply:
Awaiting for @ashutoshsharmasf review.